changelogs/fragments/1162-do_not_update_ipmi_options.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - zabbix_host - Don't reset IPMI setting when update inventory
```

Comment:
- zabbix_host - Don't reset IPMI setting when update inventory data of a host

Reply:
Thanks. I have changed to the sentence which you wrote.